src/testFixtures/java/org/opensearch/knn/ODFERestTestCase.java

```diff
@@ -212,6 +213,7 @@ private boolean skipDeleteIndex(String indexName) {
         return indexName == null
             || OPENDISTRO_SECURITY.equals(indexName)
             || IMMUTABLE_INDEX_PREFIXES.stream().anyMatch(indexName::startsWith)
```

Comment:
Why not add to immutable prefix list?

Reply:
@jmazanec15 Do you mean to add `.plugins-ml` as OPENSEARCH_ML_PLUGIN_PREFIX for `.plugins-ml-config`? If add as prefix?

Reply:
Checked on the index names in ml plugin [CommonValue.java](https://github.com/opensearch-project/ml-commons/blob/main/common/src/main/java/org/opensearch/ml/common/CommonValue.java), all indexes prefix with `.plugins-ml`

Reply:
Right

Reply:
Updated